powerauth-backend-tests/README.md

```diff
@@ -29,7 +29,7 @@ Clone the following repositories and use the default `develop` branch to build t
 
 | Repository | Command | Result |
 |---|---|---|
-| https://github.com/wultra/lime-java-core | `mvn clean install` | the libraries are installed into local maven repository (.m2 folder) |
+| https://github.com/wultra/java-core | `mvn clean install` | the libraries are installed into local maven repository (.m2 folder) |
```

Comment:
🤔 I believe that we may remove this section completely later on since we are publishing snaphosts regularly.